test/unit/specs/store/delegation.spec.js

```diff
@@ -255,4 +256,50 @@ describe(`Module: Delegations`, () => {
     await store.dispatch(`updateDelegates`)
     expect(store.state.delegation.committedDelegates).toBeTruthy()
   })
+
+  it.only(`should update the atoms on a delegation optimistically`, async () => {
```

Comment:
LOL @jbibla @NodeGuy check this new strategy to pass all unit tests

Reply:
Genius!